android-wave-recorder/src/main/java/com/github/squti/androidwaverecorder/WaveRecorder.kt

```diff
@@ -160,6 +160,17 @@ class WaveRecorder(private var filePath: String) {
         return shortData.max()?.toInt() ?: 0
     }
 
+    /**
+     * Changes @property filePath to @param newFilePath
+     * Calling this method while still recording throws an IllegalStateException
+     */
+    fun changeFilePath(newFilePath: String) {
+        if (isRecording)
+            throw IllegalStateException("Cannot change filePath when still recording. Stop recording first then call this function.")
```

Comment:
Please change the entire message to "Cannot change the file path during recording"